tutorials/haskell/hspec-golden-test-tutorial/code/LICENSE

```diff
@@ -0,0 +1,30 @@
+Copyright Author name here (c) 2019
```

Comment:
@Jagl257 Could you please add copyright name?

Reply:
Done!